extension_schema.go

```diff
@@ -20,34 +22,57 @@ type ExtensionSchema struct {
 	URL         string   `json:"url"`
 }
 
+// ListExtensionSchemaResponse is the object presented in response to the
+// request to list all extension schemas.
 type ListExtensionSchemaResponse struct {
 	APIListObject
 	ExtensionSchemas []ExtensionSchema `json:"extension_schemas"`
 }
 
+// ListExtensionSchemaOptions are the options to send with the
+// ListExtensionSchema reques(s).
 type ListExtensionSchemaOptions struct {
 	APIListObject
 	Query string `url:"query,omitempty"`
```

Comment:
@stmcallister I don't see this field documented [here](https://developer.pagerduty.com/api-reference/reference/REST/openapiv3.json/paths/~1extension_schemas/get). Does this look like a candidate for removal as part of 1.5, or should this remain?

Reply:
Although, now that I look more maybe it's implicitly supported so that you can get the total count as part of the paginated response?

Reply:
I've reached out to our engineering team to see if we can document the `query` query string parameter.

Reply:
@stmcallister since this something we can (and should) solve outside of this PR, are we good to merge this one to call the `context.Context` addition done?